### PR TITLE
chubbyphp-roadrunner

### PR DESCRIPTION
**Pull request by dominikzogg**
chubbyphp-roadrunner is same as chubbyphp but using roadrunner instead of php-fpm.

### CODE REVIEW

File: php/chubbyphp-roadrunner/config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+framework:
+  github: chubbyphp/chubbyphp-framework
+  version: 2.8
+
+before_command:
+  - vendor/bin/rr get-binary
```
Comment (waghanza) [inline php/chubbyphp-roadrunner/config.yaml:6]: @dominikzogg How can the command be different from `spiral` ?
https://github.com/the-benchmarker/web-frameworks/blob/cb5bd67bddd75db632828bb72f79d90d5347a013/php/spiral/config.yaml#L9

BTW, this is working, I have just curiosity in mind

Comment (dominikzogg) [reply php/chubbyphp-roadrunner/config.yaml:6]: @waghanza spiral got some wrapper code, i use the plain roadrunner as suggested within the roadrunner documentation

Comment (waghanza) [reply php/chubbyphp-roadrunner/config.yaml:6]: :ok_hand:

**Comment by waghanza** [comment]
@dominikzogg I have updated results

**Pull request merged**